pkg/backoff/backoff.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 const (
-	// resetAfter sets the total seconds a backoff will last.
-	resetAfter = 3600
+	// maxBackoff sets the maximum seconds we will rate limit before forcing to resend.
+	maxBackoff = 3600
```

Comment:
Lets suffix it with "Secs" to give context?
```suggestion
	maxBackoffSecs = 3600
```